interp/src/debugger/commands.rs

```diff
@@ -188,7 +189,7 @@ pub enum Command {
         Option<PrintCode>,
         PrintMode,
     ),
-    PrintPC,
+    PrintPC(Option<bool>),
```

Comment:
Please consider adding a quick comment here describing what the parameter is here. What are the meanings of the three possible values, i.e., `None`, `Some(true)`, and `Some(false)`?